README.md

```diff
@@ -2,7 +2,16 @@
 
 # Course Planner
 
-To start your Phoenix app:
+## Development Setup
+
+### Docker
+
+  * Install Elixir dependencies with `docker-compose run web mix deps.get`
+  * Setup your database (create, migrate and seed) with `docker-compose run web mix ecto.setup`
+  * Install Node.js dependencies with `docker-compose run web npm install`
```

Comment:
I agree with @gdotdesign here, these should be build steps of the image (and they are already). If you change the dependencies you should do `docker-compose build` and docker should pick the changed file and rebuild from that layer.

If i understand it correctly that these deps are getting installed into the project's dir which gets overridden on volume mount. Is it possible to adjust the target directory of dependencies to something else?

edit: and i am not exactly sure how but this is not a problem with npm in `dh_alfred` image however i assume it uses the `node_modules` subdir. or i had the libs installed locally in the project folder too 😂  i can't recall